CondFormats/Common/data/SiStripApvGainRcdAfterAbortGap_prod.json

```diff
@@ -1,5 +1,5 @@
 {
-    "destinationDatabase": "oracle://cms_orcoff_prep/CMS_CONDITIONS", 
+    "destinationDatabase": "oracle://cms_orcon_prod/CMS_CONDITIONS", 
```

Comment:
mostly for the record, this file is now obsolete (since about 2016 :)), because the corresponding `ALCAPROMPT` name was shortened to SiStripApvGainAAG to fit the DBS database limitations, so this file could be deleted at the next update (and concurrently the DropboxMetaData tag in GlobalTag as well)

Reply:
Will do, made an issue about it: https://github.com/cms-AlCaDB/AlCaTools/issues/77